### PR TITLE
Refactor logging-operator chart.

### PR DESCRIPTION
**Pull request by kozmagabor**
- autogenerated sharedKey
 - add readme
 - fix helm lint
 - rename files
 - add missing labels

### CODE REVIEW

File: logging-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+
+# Productinfo Chart
```
Comment (bonifaido) [inline logging-operator/README.md]: I think the name is bad.

File: logging-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+[Logging Operator](https://github.com/banzaicloud/logging-operator) Managed centralized logging component fluentd and fluent-bit instance on cluster.
+## tl;dr:
+
+```bash
+$ helm repo add banzaicloud-stable http://kubernetes-charts.banzaicloud.com/branch/master
+$ helm repo update
+$ helm install banzaicloud-stable/productinfo
```
Comment (bonifaido) [inline logging-operator/README.md]: And here as well.

Comment (kozmagabor) [reply logging-operator/README.md]: @bonifaido thx fixed.

File: logging-operator/README.md (hunk 1 above)
Comment (kozmagabor) [reply logging-operator/README.md]: @bonifaido thx fixed.

File: logging-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The following tables lists the configurable parameters of the logging-operator chart and their default values.
+
+|          Parameter          |                Description                               |             Default            |
+| --------------------------- | -------------------------------------------------------- | ------------------------------ |
+| `image.repository`          | Container image repository                               | `banzaicloud/logging-operator` |
+| `image.tag       `          | Container image tag                                      | `latest`                       |
+| `image.pullPolicy`          | Container pull policy                                    | `Always`                       |
+| `tls.enabled`               | Enabled TLS communication between components             | true                           |
+| `tls.secretName`            | Specified secret name, which contain tls certs           | ""                             |
```
Comment (tarokkk) [inline logging-operator/README.md]: This will overwrite automatic Helm certificate generation.

**Pull request merged**